docs/en/ingest-management/migrate-beats-to-agent.asciidoc

```diff
@@ -0,0 +1,513 @@
+*filebeat* and *metricbeat* in the list. After migrating to {agent}, you'll see
+polices named *logs* and *metrics*, which encapsulate the ILM policies for all
+`logs-*` and `metrics-*` index templates.
+
+When you migrate from {beats} to {agent}, you have a couple of options for
+migrating index policy settings:
+
+* *Modify the newly created index lifecycle policies (recommended).* As
+mentioned earlier, ILM is enabled by default when the {agent} is installed.
+Index lifecycle policies are created and added to the index templates for
+data streams created by integrations.
++
+If you have existing index lifecycle policies for {beats}, it's highly
+recommended that you modify the lifecycle policies for {agent} to match your
+previous policy. To do this:
++
+--
+. In {kib}, go to *Stack Management > Index Lifecycle Policies* and search for a
+{beats} policy, for example, *filebeat*. Under *Linked indices*, notice you can
+view indices linked to the policy. Click the policy name to see the settings.
+
+. Click the *logs* policy and, if necessary, edit the settings to match the old
+policy.
+
+. Under *Index Lifecycle Policies*, search for another {beats} policies, for
+example, *metricbeat*.
+
+. Click the *metrics* policy and edit the settings to match the old policy.
+--
++
+Optionally delete the {beats} index lifecycle policies when they are no longer
+used by an index.
+
+* *Keep the {beats} policy and apply it to the index templates created for data
+streams.* To preserve an existing policy, modify it, as needed, and apply it to
+all the index templates created for data streams:
++
+--
+. Under *Index Lifecycle Policies*, find the {beats} policy, for example,
+*filebeat*.
+
+. In the *Actions* column, click the *Add policy to index template* icon.
+
+. Under *Index template*, choose a data stream index template, then add the
+policy.
+
+. Repeat this procedure, as required, to apply the policy to other data stream
+index templates.
+--
+
+//QUESTION: The UI gives you the option of setting the rollover alias. Should
+//we say anything about that here, or just assume they'll leave it blank?
+
+.What if you're not using ILM with {beats}?
+****
+You can begin to use ILM now with {agent}. Under *Index lifecycle policies*,
+click the policy name and edit the settings to meet your requirements.
+****
+
+[discrete]
+== Remove {beats} from your host
+
+Any installed {beats} shippers will continue to work until you remove them. This
+allows you to roll out the migration in stages. You will continue to see
+duplicated data until you remove {beats}.
+
+When you're satisfied with your {agent} deployment, remove {beats} from your
+hosts. All the data collected before installing the {agent} will still be
+available in {es} until you delete the data or it's removed according to the
+data retention policies you've specified for ILM.
+
+To remove {beats} from your host:
+
+//REVIEWERS: I'm just making stuff up below to show you the types of info I think
```

Comment:
i agree that this is required. Just checked and you are right we don;t have anything on beats.
Should this be added to the beats and then referenced from here?  beats just talks about how to stop the beat.

Reply:
Yah I think so, but let's get this merged as it is, and I'll open an issue to get something added to the Beats documentation at some point. We have a pretty big backlog of Beats doc issues  right now (for reasons that we can discuss offline if you'd like). Since no one has complained about this info being missing, I suspect that most Beats users have figured this out on their own, so maybe it's not a high priority. I think in the context of replacing Beats with Agent, though, removing Beats is more critical.

Reply:
Here's the issue: https://github.com/elastic/beats/issues/30387